Code/lib/corner/corner.cpp

```diff
@@ -0,0 +1,39 @@
+#include <corner.h>
+
+CornerMerchantry::CornerMerchantry(float *right_threshold, float *front_threshold){
+    set_threshold_right = right_threshold;
+    set_threshold_front = front_threshold;
+}
+
+bool CornerMerchantry::wall_detected(float distance, float threshold) {
```

Comment:
Why can distance and threshold be private variables?

Reply:
only used by the class and not required elsewhere

Reply:
I see. I have seen the distance variable is the one that changes but threshold is constant

Reply:
What about something like this
```cpp
bool CornerMerchantry::wall_detected(float distance)
```

Reply:
Side and front threshold may be different hence the need for the 2nd parameter cause the function is called twice